src/lang/std/artifactGraph.test.ts

```diff
@@ -247,7 +247,7 @@ describe('testing createArtifactGraph', () => {
       // of the edges refers to a non-existent node, the graph will throw.
       // further more we can check that each edge is bi-directional, if it's not
       // by checking the arrow heads going both ways, on the graph.
-      await GraphTheGraph(theMap, 1400, 1400, 'exampleCode1.png')
+      await GraphTheGraph(theMap, 2000, 2000, 'exampleCode1.png')
```

Comment:
There are more nodes in the graph now, so bumping the resolution of the captured image